### ISSUE TITLE
A question about navigation

### ISSUE THREAD

**Issue by kseistrup** [xvxx/phetch#3]
When looking at the `keyboard shortcuts` help page in `phetch`, I can't help wondering: Is there a reason we can't lose the `Ctrl-` part of many of the commands?

If we use the left/right/up/down arrows to move the cursor, and use numbers (or `Enter`) to select a link, couldn't we just choose `g` for “go to”, `b` for “bookmarks”, and so on: i.e., losing the `Ctrl-` part entirely (except for `^C`, perhaps), thus only having to press a single key in order to indicate an action?

**Comment by xvxx** [comment]
Yes, I think that's probably the more obvious way to do things. But re-reading that help page, I think I left out the most important keyboard shortcut: you are in search mode by default. I tend to use this a lot, typing a link's name to jump to it and moving between matches with `ctrl-n` and `ctrl-p` instead of arrow keys. 

But `vim` and `lynx` do it the way you suggest, with `/` to trigger the incremental search and single characters representing other commands. So right now I am thinking that I'll change it so you're in "command mode" by default, where you can just type `g` or `s` like you suggest to activate commands, and you'll have to type `/` to enter search mode.

(Then probably a config file or something that lets you launch into search mode if you want.)

**Comment by kseistrup** [comment]
> search mode

Ah, so _that_ is why characters tend to pile up in the lower left corner. I didn't realise it was search mode. I feel a little sheepish now... :wink:

But I like your idea of being in command mode by default and then using `/` to enter search mode. That would be easy to remember.

**Comment by xvxx** [comment]
No no, the UI shouldn't be a puzzle. It's my fault for making it so non-obvious. 

I've switched it so it's in "command mode" by default now, so `g` will go to a url and help is `h`. But `ctrl-` also works, so `ctrl-g` and `ctrl-h` do the same things. 

For the search, `/` or `f`/`ctrl-f` will launch the incremental searcher now. I think it makes a lot more sense and is easier to explain this way, since now the keyboard shortcuts page tells more of the whole picture. 

I just pushed out [v0.1.3](https://github.com/dvkt/phetch/releases/tag/v0.1.3) with these changes, let me know if you run into any issues or have any more ideas! Thanks again!
